gofaxd/server.go

```diff
@@ -221,6 +221,7 @@ func (e *EventSocketServer) handler(c *eventsocket.Connection) {
 	}
 	c.Execute("set", fmt.Sprintf("fax_ident=%s", csi), true)
 	c.Execute("rxfax", filenameAbs, true)
+	c.Execute("hangup", "", false)
```

Comment:
I'm not sure though if the parameter `lock` which I set to `false` should be `true` instead...